src/components/contribution/ContributionAmount.vue

```diff
@@ -116,7 +118,9 @@ const hasError = computed(() => validation.value.$errors.length > 0);
 
 const rules = computed(() => ({
   amount: {
-    minValue: minValue(toRefs(props).minAmount),
+    ...(!props.disabled && {
+      minValue: minValue(toRefs(props).minAmount),
+    }),
   },
 }));
```

Comment:
I find this difficult to read, why not instead like this?

```ts
cconst ruls = computed(() => {
  if (props.disabled) return { amount: {} };

  return {
    amount: {
      minValue: minValue(toRefs(props).minAmount),
    },
  };
});
```

Reply:
I'm definitely guilty of doing this sort of thing way too much :+1:

What do you think of using a ternary operator, e.g.
```ts
const rules = computed(() =>
  props.disabled
    ? { amount: {} }
    : {
        amount: {
          minValue: minValue(toRefs(props).minAmount),
        },
      }
);
```

I find ternary operators easy to read and offer less potential for future developers to do strange things with not seeing early returns etc.

Reply:
Looks good 👍